web/pdf_find_controller.js

```diff
@@ -489,6 +497,7 @@ class PDFFindController {
     this._pdfDocument = null;
     this._pageMatches = [];
     this._pageMatchesLength = [];
+    this.#visitedPagesCount = 0;
```

Comment:
This isn't sufficient on its own, since you must *also* reset it when e.g. the active search term changes; hence this line is also necessary after https://github.com/mozilla/pdf.js/blob/8a0ca04399cd1b4b4d1bb960685550e3f6ea4e03/web/pdf_find_controller.js#L839-L840